_posts/2019-12-20-this-blog-post-contains-important-information-for-some-percentage-of-its-readers.markdown

```diff
@@ -0,0 +1,150 @@
+then something will break because of it and it's very important not to yell at them about it.
+
+You're used to modeling the world in a specific way, but school doesn't actually prepare you in any way for the
+complex web of social dynamics that, just like in kindergarten, everyone else seems to grok. It's really
+frustrating, maybe you've been fired once or twice, maybe you've left a few jobs out of embarrassment. You
+eventually figured it out -- or maybe you didn't! The thing is, programming has never been the hard part of work,
+for you. The people have been the much more interesting and challenging domain to master.
+
+Today, you're an engineer who really deeply understands and enjoys your stack. You have workflows that you can do
+in your sleep, you know intuitively what the current state of your system is and you have a powerful intuition for
+downstream problems. Maybe you've figured out how to play the social game, in which case you're probably gregarious
+and supportive and kind to your coworkers. Maybe you haven't, in which case you're super productive but don't
+attend a lot of office functions. Either way is fine as long as you've found a place where you feel comfortable!
+
+At the same time, you've had a sense for years that something isn't quite... right. You've probably learned to
+ignore it. Sure things get super stressful sometimes. Sure you find yourself really unreasonably tired, especially
+after meeting-heavy days. Sure when you get downtime sometimes all you can do is just sit there doing nothing --
+maybe you feel bad about that? Maybe you use an abnormally high number of sick days purely for mental health
+reasons. Maybe you've got a substance abuse issue. You're probably struggling with codependency, anxiety and/or
+depression. You think of yourself as just "a depressed person", but try not to let it define you. You feel
+constantly on the edge of burnout, but no matter how many vacations you take it doesn't seem to get any better.
+
+Maybe you are sensitive to certain kinds of sensory stimuli - like, you can't wear synthetic fabric because it just
+feels gross. Maybe you can't eat peaches because the peach fuzz on the roof of your mouth is unbearable. Maybe you
+can't go into a well-lit room without sunglasses, maybe every sound you hear comes in at the same volume. Maybe a
+bunch of these kinds of things or more bother you, and maybe you've forgotten because you've just pushed through
+them your whole life. Maybe you've accepted that you're just kind of weird and different.
+
+But maybe -- and this may be the best news you've ever received -- you're autistic.
+
+Many if not most autistic people would tell you similar stories about their personal histories, especially those
+who have been undiagnosed into adulthood. If these descriptions resonate with you -- maybe not exactly, but maybe
+you can see the shape I'm trying to paint -- then it's worth considering the possibility that you may be
+neurodivergent.
```

Comment:
This is a really key part of the post – I feel like it could be emphasized with either **bold text**, or even better, with a link out to further reading.